### PR TITLE
Bug 1134961 - Make suggestion bar and ICE Contacts rtl compatible. r=gsv...

### PR DESCRIPTION
**Pull request by tamarahills**
...elto.

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ae353ac056fd92cc119c235ade1c645ed6485e20

File: apps/communications/dialer/style/suggestion.css [hunk 1]
```diff
@@ -7,6 +7,12 @@
   width: 100%;
   color: #033B37;
 }
+html[dir="ltr"] .suggestion-bar {
```
Comment (DouglasSherk) [inline apps/communications/dialer/style/suggestion.css]: Interesting pattern of grouping all ```[dir=***]``` keys together with their general counterparts. Our pattern so far has been to space them out more, though, so let's stick with that.

File: apps/communications/dialer/style/suggestion.css [hunk 2]
```diff
@@ -7,6 +7,12 @@
...
+  margin-left: -1.5rem;
```
Comment (DouglasSherk) [inline apps/communications/dialer/style/suggestion.css]: Why do these need different numbers? We should figure out why, and if possible, use ```margin-start: -1.5rem``` or some such, instead.

File: shared/style/contacts/contact_in_overlay.css [hunk 3]
```diff
@@ -46,11 +50,42 @@
...
+}
+
+.ellipsis-name {
+  display: inline-block;
+  width: 100%;
   text-overflow: ellipsis;
   overflow: hidden;
-  margin: 0 auto;
+  white-space: nowrap;
+  margin-bottom: -.3rem;
```
Comment (DouglasSherk) [inline shared/style/contacts/contact_in_overlay.css]: Write this as:
```css
margin-bottom: -0.3rem;
```

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e179dc16e5a7d9658a915a2dffa6c4a101394e22

**Pull request closed without merging**